soc/nxp/imxrt/soc.yml

```diff
@@ -34,3 +34,69 @@ family:
     - name: mimxrt685s
       cpuclusters:
       - name: cm33
+runners:
```

Comment:
It looks like we could use regex to simplify this list, based off how the issue was handled for nordic SOCs: https://github.com/zephyrproject-rtos/zephyr/pull/69748/files#diff-a8f951df972bcad685b06e6cd38a29a415ceff396a22dbc19d65f8a46eb0e590R54?

Reply:
> It looks like we could use regex to simplify this list, based off how the issue was handled for nordic SOCs: https://github.com/zephyrproject-rtos/zephyr/pull/69748/files#diff-a8f951df972bcad685b06e6cd38a29a415ceff396a22dbc19d65f8a46eb0e590R54?

Added for rt10xx

Reply:
Nice- could we do something as basic as `- mimxrt.*` for the match? I'm not sure if this is too broad, or even valid syntax for the YAML match here- just a thought, we could probably do the same for the Kinetis and LPC with `mk.*` and `lpc.*` if this is a valid syntax to use

Reply:
> Nice- could we do something as basic as `- mimxrt.*` for the match? I'm not sure if this is too broad, or even valid syntax for the YAML match here- just a thought, we could probably do the same for the Kinetis and LPC with `mk.*` and `lpc.*` if this is a valid syntax to use

Even Nordic is not so brave ;) 
I can do experiments only in 2 weeks, after vacation.